### PR TITLE
MINOR:Fix hint about alter in TopicCommand

### PR DESCRIPTION
**Pull request by hudeqi**
Since TopicCommand does not support updating the configuration of an existing topic, "and/or configuration for the topic." should be removed from the description of --alter, otherwise it will cause misunderstandings for those users of the old kafka version.

### CODE REVIEW

**Comment by hudeqi** [comment]
Anyone take a look？

File: core/src/main/scala/kafka/admin/TopicCommand.scala
Comment (dengziming) [inline core/src/main/scala/kafka/admin/TopicCommand.scala:1]: Thank you for this PR, There are some fields/methods in `CommandTopicPartition` which is also unused, can you also remove them.

Comment (hudeqi) [reply core/src/main/scala/kafka/admin/TopicCommand.scala:1]: Ok, I probably have deleted some useless code, please help to review again, thank you!

**Comment by hudeqi** [comment]
Can you help review to see if there is anything else that needs to be removed? @dengziming

**Review by dengziming — APPROVED** [review]
LGTM.

**Pull request merged**